components/private/Math/tests/unit/MDCMathTests.m

```diff
@@ -23,6 +23,32 @@ @interface MDCMathTests : XCTestCase
 
 @implementation MDCMathTests
 
+#pragma mark - MDCCeilScaled and MDCFloorScaled
+
+- (void)testMDCFloorScaled {
```

Comment:
Please add more tests and structure them in the Given-When-Then style that we try to encourage in general. We prefer to have 1 test for each test condition whenever possible.

Reply:
Please add tests for negative values as well.

Overall looks good!